OpenTabletDriver.Desktop/Binding/BindingState.cs

```diff
@@ -23,7 +23,8 @@ public virtual void Invoke(TabletReference tablet, IDeviceReport report, bool ne
 
             if (Binding is IInterruptBinding interruptBinding)
             {
-                interruptBinding.Invoke(tablet, report);
+                if ((newState && !PreviousState) || (!newState && PreviousState))
```

Comment:
Won't something like this be enough?
```suggestion
                if (newState != PreviousState)
```